### DISCUSSION TITLE
[Idea/Feature] Manage content in partner’s library

### DISCUSSION THREAD

**Discussion by l13t** [immich-app/immich#7029] [Feature Request]
Maybe I missed and there were already discussions on this, so I’ll be happy if you point me to a discussion.
Anyway, I’m using Immich as a library for our family photos. I have an iOS app installed on my wife’s and my phones. Sometimes we both take photos of garbage that should not pop up in the family album: shelves in a supermarket, stuff we sell, some accidentally made photos, etc. For me it’s easy - I clean up periodically my part of the photos from such types of garbage. But my wife doesn’t do this. I can obviously log in with her credentials and remove what we don’t need. But it would be great to have the possibility to add some flag on the library sharing page that allows partners to remove photos of each other. Or maybe some global option that allows the admin user to do this.

**Comment by maggu2810** [comment]
While looking for other comments like that I found yours and the issue transformed to a discussion here: #9441
Perhaps a duplicate now.
